website/src/css/custom.css

```diff
@@ -0,0 +1,2005 @@
+:root {
```

Comment:
This looks like the entire vendor file - is there a way to keep the default base stylesheet separate from our customizations? Otherwise, it will be difficult to update when the base file changes.

Reply:
Yeah, I haven't finished the CSS stuff, it's the next thing I'm planning to tackle :)